plugins/env/app/decorators/stage_decorator.rb

```diff
@@ -0,0 +1,6 @@
+Stage.class_eval do
+  has_many :stage_environment_variable_groups
+  has_many :environment_variable_groups, through: :stage_environment_variable_groups
```

Comment:
You don't have to specify the through table if you use the HABTM relationship. See DeployGroups to Stages
```
  has_and_belongs_to_many :environment_variable_groups
```